cmd/admin-rpc-server.go

```diff
@@ -18,223 +18,110 @@ package cmd
 
 import (
 	"context"
-	"encoding/json"
-	"fmt"
-	"io/ioutil"
-	"os"
-	"path/filepath"
+	"path"
 	"time"
 
 	"github.com/gorilla/mux"
 	"github.com/minio/minio/cmd/logger"
+	xrpc "github.com/minio/minio/cmd/rpc"
 )
 
-const adminPath = "/admin"
+const adminServiceName = "Admin"
+const adminServiceSubPath = "/admin"
 
-// adminCmd - exports RPC methods for service status, stop and
-// restart commands.
-type adminCmd struct {
-	AuthRPCServer
+var adminServicePath = path.Join(minioReservedBucketPath, peerServiceSubPath)
+
+// adminRPCReceiver - Admin RPC receiver for admin RPC server.
+type adminRPCReceiver struct {
```

Comment:
adminRPCServer is a better name?

Reply:
Based on Go RPC terms, I used receiver than server.  At server side, two components required at server side.  Receiver type contains methods for RPCs and RPC server receives/serves RPC request/response.  The server contains one or more receivers (not confirmed/tested with 2+ receivers) to serve RPCs. Hence both are two different thing.

Reply:
Got it. But...

`AdminRPCClient` and `AdminRPCServer` looks complementary and predictable names.

`AdminRPCClient` and `AdminRPCReceiver` does not sound complementary.

Since we are not using Go RPC anymore, we could use the names that are more meaningful for us.

> The server contains one or more receivers (not confirmed/tested with 2+ receivers) to serve RPCs. Hence both are two different thing.

For the RPC package, it can think of `adminRPCServer` as a sub-rpc-server instead of an unfamiliar term like `receiver`